ecl/hql/hqldesc.cpp

```diff
@@ -90,9 +109,6 @@ static void expandRecordSymbolsMeta(IPropertyTree * metaTree, IHqlExpression * r
             {
                 IPropertyTree * field = metaTree->addPropTree("Field");
                 field->setProp("@name", str(cur->queryId()));
-                StringBuffer ecltype;
```

Comment:
why remove the type of the field?

To correctly represent record types may require more though to prevent the file getting too large.